backend/src/ml_space_lambda/initial_app_config/lambda_function.py

```diff
@@ -46,6 +47,7 @@ def lambda_handler(event, context):
     ]
 
     app_configuration_dao.update(config)
+    update_instance_constraint_policies(config, context)
```

Comment:
Just checking, but what if DR isn't enabled, will this be able to add the required policies to the allowed instance types? 

If the answer is no, I suppose the customer just needs to make sure they have their IAM policies sorted or correctly right?

Reply:
This won't be able to update if DR is enabled. I am going to update this with a condition to check that for good measure.

If they're not using DR it is up to them to have their policies in sync with the UI. Likely they'll just have permissive policies though.